text/0002-searchable-display-attributes-behaviors.md

```diff
@@ -0,0 +1,123 @@
+- Title: Searchable Attributes and Displayed Attributes behaviors
+- Start Date: 2020-11-02
+- specification PR: meilisearch/specifications#3
+- Meilisearch Issue: 
+
+# `searchable_attributes` & `displayed_attributes` behaviors
+
+## First section: Feature Description and Interaction
+
+This first part has a general audience. It should be as little technical as possible (think user-level). This section contains 4 sub-sections:
+
+### Summary
+
+Ever since the beginning, the design of `searchableAttributes` and `displayedAttributes` as shown flaws. The introduction of the wildcard tried to adress some of those flaws, but there are still issues that this specs tries to correct.
+
+Searchable attributes have several uses:
+- know in which fields we can search.
+- know the importance of fields for the criterion `Attributes`.
+
+The main issues we want to adress here are:
+- The question of `searchableAttributes` order when the wildcard is used.
+- The fact that already known attributes (e.g from facets), are not indexed correctly is put before the documents (bug), need to be taken into account in the design.
+
+Is it relevant to mix this 2 information when we have wildcards?
+
+### Explanation
+
+#### default values
+
+`searchable_attributes`: *   
+`displayed_attributes`: *
```

Comment:
We should choose a consistent syntax: `searchableAttributes` or `searchable_attributes` :)